cli/Cargo.toml

```
@@ -14,9 +14,12 @@ name = "jstime"
 path = "main.rs"
 
 [dependencies]
-jstime_core = { path = "../core", version = "^0.12.5-alpha.0"}
+jstime_core = { path = "../core" }
 rustyline = "6.2.0"
 structopt = "0.3.17"
 
+[build-dependencies]
+jstime_core = { path = "../core" }
```

```auggestion
```suggestion
jstime_core = { path = "../core", version = "^0.12.5-alpha.0"}
```

I'm not 100% (haven't tested just yet, but my gut is we need to be explicit here too